src/content/en/fundamentals/performance/rail.md

```diff
@@ -67,12 +75,34 @@ Understand how users perceive performance delays:
   </tbody>
 </table>
 
-## Response: respond in under 100ms {: #response }
+The perception of performance delays can vary based on mobile hardware
+and network conditions. While loading an experience in 1000ms is plausible on
+a powerful desktop machine over cable, on mobile, ensuring users can load and 
+interact with content in under 5000s is a more [realistic](http://chimera.labs.oreilly.com/books/1230000000545/ch08.html#MOBILE_NETWORK_LATENCY_OVERHEAD) target under slow 3G.
```

Comment:
@addyosmani two points (no action needed on your part)

1. this URL redirects to https://hpbn.co/optimizing-for-mobile-networks/ is this the same content as what you inteded?
2. Our style guide recommends descriptive link texts so readers have a better idea of what they're opening https://developers.google.com/style/link-text